pkg/yaml2pcl/yaml2pcl.go

```diff
@@ -454,7 +456,7 @@ func walkToPCL(v Visitor, node ast.Node, totalPCL io.Writer, suffix string) erro
 				return err
 			}
 			if value.Type() == ast.MappingValueType {
-				_, err = fmt.Fprintf(totalPCL, "%s%s\n", "}", "")
+				_, err = fmt.Fprintf(totalPCL, "%s%s\n", "}", suffix)
```

Comment:
Why not just do this?
```suggestion
				_, err = fmt.Fprintf(totalPCL, "}%s\n", suffix)
```

Reply:
I'm just opting to maintain the same code style we have elsewhere in this file.